module/checker.go

```diff
@@ -14,6 +14,17 @@ import (
 	"github.com/lyft/protoc-gen-star"
 )
 
+var unknown = ""
+var httpHeaderName = "^:?[0-9a-zA-Z!#$%&'*+-.^_|~\x2C]+$"
```

Comment:
Is there a missing backtick here?Where does the comma (0x2c) come from?

Reply:
Oops I think I mixed them up somewhere along the way. Backtick (0x60) in place of comma now. But summarized field-name = token = ALPHANUM / these allowables